src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ReviewTimeExtensionConfirmation.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.postsubmit;
+
+import static java.util.Objects.requireNonNull;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PostSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iacaseapi.domain.handlers.PostSubmitCallbackHandler;
+
+@Component
+public class ReviewTimeExtensionConfirmation implements PostSubmitCallbackHandler<AsylumCase> {
+
+    public boolean canHandle(
+            Callback<AsylumCase> callback
+    ) {
+        requireNonNull(callback, "callback must not be null");
+
+        return callback.getEvent() == Event.REVIEW_TIME_EXTENSION;
+    }
+
+    public PostSubmitCallbackResponse handle(
+            Callback<AsylumCase> callback
+    ) {
+        if (!canHandle(callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+
+        PostSubmitCallbackResponse postSubmitResponse =
+                new PostSubmitCallbackResponse();
+
+
+        postSubmitResponse.setConfirmationHeader("# You have been granted a time extension");
+        postSubmitResponse.setConfirmationBody(
+                "#### Do this next\n"
+                        + "You must change the direction due date in the directions tab\n"
+        );
+
```

Comment:
This isn't correct now we should be doing this as part of the this event so don't need to add this